src/app/Dashboard/AutomatedAnalysis/AutomatedAnalysisFilters.tsx

```diff
@@ -192,7 +199,7 @@ export const filterAutomatedAnalysis = (
 
   let filtered = topicEvalTuple;
 
-  if (!!filters.Name.length) {
+  if (filters.Name.length) {
```

Comment:
Somewhere in this function `filterAutomatedAnalysis` is still using `!!` and `!= null` (should be checked against `undefined` or using short cut as here).

Reply:
Whoops, I don't remember why I changed this...

Reply:
Oh, it was automatically "fixed" :
```
error  Redundant double negation  no-extra-boolean-cast
```